pytripgui/view/qt_view_adapter.py

```diff
@@ -172,6 +172,14 @@ def checked_items(self):
                 selected.append(widget.data(Qt.UserRole))
         return selected
 
+    def check_items(self, items, lambda_names):
+        to_be_checked = [
+            list_item for list_item in self._items
+            if lambda_names(list_item.data(Qt.UserRole)) in [lambda_names(item) for item in items]
```

Comment:
what is lambda_names ?

Reply:
copied from another method signature - lambda that gets name from passed items

Reply:
it is not unclear, what about `name_extractor` ? Are we forced to use lambdas here, I suspect function will work as well

Reply:
Done